EasyRestProjectNetTeam2/EasyRestPages/ManageRestaurantsPage.cs

```diff
@@ -0,0 +1,90 @@
+﻿using OpenQA.Selenium;
+using SeleniumExtras.PageObjects;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace EasyRestProjectNetTeam2.EasyRestPages
+{
+    public class ManageRestaurantsPage : BasePage
+    {
+        public ManageRestaurantsPage(IWebDriver driver) : base(driver) { }
+
+        private const int TimeToWait = 20;
```

Comment:
don't use this in the page object, this is what you need to pass into the page object from the test

Reply:
Fixed